### PR TITLE
Update to Ambassador 1.5.0

### PR DESCRIPTION
**Pull request by kflynn**
Update to Ambassador 1.5.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ numbering uses [semantic versioning](http://semver.org).
 - Add option to install Service Preview components (traffic-manager, traffic-agent).
 - Add option to install ambassador-injector, alongside Service Preview.
 
+## v6.4.1
```
Comment (alexgervais) [inline CHANGELOG.md:13]: A few lines up, there are changelog items for "Next Release". These should be moved under v6.4.1.

Comment (nbkrause) [reply CHANGELOG.md:13]: *sigh* this should actually be under 6.4.0.

That release was never actually pushed and I updated the changelog incorrectly.

File: Chart.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 apiVersion: v1
-appVersion: 1.4.3
-ossVersion: 1.4.3
+appVersion: 1.5.0
+ossVersion: 1.5.0
 description: A Helm chart for Datawire Ambassador
 name: ambassador
-version: 6.3.6
+version: 6.4.1
```
Comment (nbkrause) [inline Chart.yaml:6]: Please change this to 6.4.0

File: CHANGELOG.md (hunk 1 above)
Comment (kflynn) [reply CHANGELOG.md]: Thanks!

File: Chart.yaml (hunk 2 above)
Comment (kflynn) [reply Chart.yaml]: Done.

**Pull request merged**